moi/test/test_group.py

```diff
@@ -144,9 +144,9 @@ def __call__(self, data):
 
     def test_action_add(self):
         resp = self.obj._action_add(['d', 'f', 'e'])
-        self.assertEqual(resp, [
+        self.assertEqual(sorted(resp), sorted([
```

Comment:
Snagged all of them

On Tue, May 19, 2015 at 10:33 AM, Antonio Gonzalez <notifications@github.com
> wrote:

> In moi/test/test_group.py
> <https://github.com/biocore/mustached-octo-ironman/pull/24#discussion_r30618463>
> :
>
> > @@ -144,9 +144,9 @@ def __call__(self, data):
> >
> >      def test_action_add(self):
> >          resp = self.obj._action_add(['d', 'f', 'e'])
> > -        self.assertEqual(resp, [
> > +        self.assertEqual(sorted(resp), sorted([
>
> And this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/mustached-octo-ironman/pull/24/files#r30618463>
> .
>